### PR TITLE
[jsk_recognition_utils] add rect_array_to_cluster_point_indices.py

### PR DESCRIPTION
**Pull request by knorth55**
add `rect_array_to_cluster_point_indices.py`
this node converts `RectArray` to `ClusterPointIndices`, which is useful for object detection nodes.

rects_to_cpi

it is related to #2467

### CODE REVIEW

File: doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  Output cluster point indices.
+
+## Parameters
+
+* `~use_info` (Bool, Default: `false`)
+
+  Subscribe camera_info topic or not
+
+* `img_width`: (Int, required)
```
Comment (iory) [inline doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md:30]: ~img_width

File: doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  Image width. Required only when `use_info` is `false`.
+
+* `img_height`: (Int, required)
```
Comment (iory) [inline doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md:34]: ~img_height

File: doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  Image height. Required only when `use_info` is `false`.
+
+* `~queue_size` (Int, Default: `10`)
+
+  Queue size for `message_filters`. Used only when `use_info` is `true`.
+
+* `approximate_sync` (Bool, Default: `false`)
```
Comment (iory) [inline doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md:42]: ~approximate_sync

File: doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+  Use approximate_sync or not. Used only when `use_info` is `true`.
+
+* `slop` (Float, Default: `0.1`)
```
Comment (iory) [inline doc/jsk_recognition_utils/nodes/rect_array_to_cluster_point_indices.md:46]: ~slop

**Comment by knorth55** [comment]
thanks, I updated.

**Comment by knorth55** [comment]
@k-okada can you check and merge this PR?

**Pull request merged**